dlk/python/dlk/templates/include/global.tpl.h

```diff
@@ -30,12 +30,32 @@ typedef int16_t   T_INT16;
 
 #define QUANTIZED_NOT_PACKED uint8_t
 
+enum class MemoryFormat {
+  HWC,
+};
+
+template <typename pack_type, MemoryFormat memory_format>
+class QuantizedPacked {
+ public:
+  using T = pack_type;
+  QuantizedPacked() = default;
+  explicit QuantizedPacked(const T val) : val(val) {}
+  explicit operator T() const { return val; }
+  explicit operator T() { return val; }
```

Comment:
having only `const` version of `operator T()` here should be enough.

```
 explicit operator T() const { return val; }
```

could be used in both `QuantizedPacked&` and `const QuantizedPacked&` cases.

Reply:
Fixed.